docs/src/main/paradox/features/additional-directives.md

```diff
@@ -0,0 +1,72 @@
+Additional directives
```

Comment:
Sorry I didn't do a better job at researching a good name. Turns out that foundation has callouts.